src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/collectionsgrid/view/CollectionsGrid.kt

```diff
@@ -0,0 +1,83 @@
+package org.wycliffeassociates.otter.jvm.app.ui.collectionsgrid.view
+
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.beans.property.Property
+import javafx.scene.layout.Priority
+import javafx.stage.Screen
+import org.wycliffeassociates.otter.common.data.model.Collection
+import org.wycliffeassociates.otter.jvm.app.theme.AppStyles
+import org.wycliffeassociates.otter.jvm.app.theme.AppTheme
+import org.wycliffeassociates.otter.jvm.app.ui.collectionsgrid.CollectionGridStyles
+import org.wycliffeassociates.otter.jvm.app.ui.collectionsgrid.viewmodel.ViewCollectionsViewModel
+import org.wycliffeassociates.otter.jvm.app.widgets.card.DefaultStyles
+import org.wycliffeassociates.otter.jvm.app.widgets.card.card
+import tornadofx.*
+
+class CollectionsGrid : Fragment() {
+    private val viewModel: ViewCollectionsViewModel by inject()
+
+    val activeCollection: Property<Collection> = viewModel.activeCollectionProperty
+    val activeProject: Property<Collection> = viewModel.activeProjectProperty
+
+    init {
+        importStylesheet<CollectionGridStyles>()
+        importStylesheet<DefaultStyles>()
+    }
+
+    override val root = stackpane {
+        hgrow = Priority.ALWAYS
+        vgrow = Priority.ALWAYS
+        style {
+            prefWidth = Screen.getPrimary().visualBounds.width.px - 50.0
```

Comment:
I don't think we want this?

Reply:
Like, it's a nested component and we have it set to hgrow/vgrow already

I assume hgrow and vgrow work here; like, they're not just for hbox/vbox, right?

Reply:
hgrow and vgrow don't seem to produce the desired effect. i'm trying to find a solution